app/controllers/application_controller.rb

```diff
@@ -120,6 +122,10 @@ def authenticate_user!
     end
   end
 
+  def set_default_view_variables
+    @adv_search = Query::Proposal::FieldedSearch.new({})
```

Comment:
it was definitely intended to make search available on every page.

If we need to re-think the UI layout to make it clear what is being searched, my 2c is let's do that.

cc: @amberwreed @phirefly